extensionsGallery.json

```diff
@@ -3603,14 +3603,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.0",
-							"lastUpdated": "10/14/2020",
+							"version": "0.3.2",
```

Comment:
I see there's already a VSIX uploaded to the azure storage but you also have a PR out that updates the extension version to 0.3.2. So where did that VSIX come from then? Did you queue a private build or something? We should only be releasing VSIXs that come from the official build pipelines for main or the release branch.

Reply:
Yes, we get it from official build usually. Private build was done this time just because I was oof last few days and we didn't get to update it timely after testing etc before the November release date.